fieldservice_fsm_person_salary_extended/views/fsm_order_salary_views.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_fsm_person_salary_extend_tree" model="ir.ui.view">
+        <field name="name">fsm.order.salary.tree (FSM Person Salary Extended)</field>
+        <field name="model">fsm.order.salary</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="sequence" widget="handle" />
+                <field name="fsm_order_id" />
+                <field name="fsm_person_id" />
+                <field name="partner_id" />
+                <field name="vehicle" />
+                <field name="payments_salary" sum="Total" />
+                <field name="payments_km" sum="Total" />
+                <field name="payments_diets" sum="Total" />
+                <field name="payments_advance" sum="Total" />
+                <field name="net_salary" sum="Total" />
+                <field name="currency_id" invisible= '1' sum="Total" />
+            </tree>
+        </field>
+    </record>
+
+
+    <record id="view_fsm_person_salary_extend_search" model="ir.ui.view">
+        <field name="name">fsm.order.salary.filter (FSM Person Salary Extended)</field>
+        <field name="model">fsm.order.salary</field>
+        <field name="arch" type="xml">
+            <search string="Order Salary">
+                <filter string="My Orders Salary" 
+                    domain="[('fsm_person_id.partner_id.user_ids', '=', uid)]" name="my_orders_salary"/>
+                <separator/>
+                <field name="fsm_person_id"/>
+                <field name="location_id"/>
+                <field name="scheduled_date_start" />
+                <separator/>
+                <filter string="Fsm Person" name="fsm_person_id" />
```

Comment:
I dont understund this filters, can there be wages without persons or locations?